plugins/org.locationtech.udig.ui.tests/src/org/locationtech/udig/internal/ui/operations/OpFilterPropertyValueConditionTest.java

```diff
@@ -0,0 +1,119 @@
+package org.locationtech.udig.internal.ui.operations;
```

Comment:
since it is a new class file we should add a file comment as defined in https://github.com/locationtech/udig-platform/blob/master/extras/org.locationtech.udig.dev/codetemplates.xml here